Work out why delay test is breaking the Travis build

**Package**

lib-react-components

**Feature or Issue Description**

While working on #487, in `packages/lib-react-components/src/Media/components/ThumbnailImage/ThumbnailImage.spec.js`, I had to skip a test as it breaks the Travis build. It runs fine locally, but on Travis it times out.

This is the failing test: https://github.com/zooniverse/front-end-monorepo/blob/4191d43775f75b7b21912d20c883b24e79a67620/packages/lib-react-components/src/Media/components/ThumbnailImage/ThumbnailImage.spec.js#L45-L55

Adding `done()` to that test fixes the test runner for me, but the test is just broken. It doesn't render an `Image` component so fails with.
```
1) Image
       should delay loading the image the given time in props.delay:

      Uncaught AssertionError: expected {} to have a length of 0 but got 1
      + expected - actual

      -1
      +0
      
      at Timeout.lengthOf [as _onTimeout] (src/Media/components/ThumbnailImage/ThumbnailImage.spec.js:53:49)
```

I think it's only passing at the moment because async tests without `done` are evergreen. See https://wietse.loves.engineering/testing-promises-with-mocha-90df8b7d2e35

--- a/packages/lib-react-components/src/Media/components/ThumbnailImage/ThumbnailImage.spec.js
+++ b/packages/lib-react-components/src/Media/components/ThumbnailImage/ThumbnailImage.spec.js
@@ -42,15 +42,16 @@ describe('Image', function () {
     expect(wrapper.find(Image)).to.have.lengthOf(0)
   })
 
-  xit('should delay loading the image the given time in props.delay', function () {
+  it('should delay loading the image the given time in props.delay', function (done) {
     const delay = 1000
     const wrapper = mount(<ThumbnailImage delay={delay} src={image} />)
     const progressiveImageInstance = wrapper.find(ProgressiveImage).instance()
     progressiveImageInstance.onLoad()
-    wrapper.update()
-    setTimeout(() => {
+    setTimeout(function () {
+      wrapper.update()
       expect(wrapper.find(Placeholder)).to.have.lengthOf(0)
       expect(wrapper.find(Image)).to.have.lengthOf(1)
+      done()
     }, delay + 1)
   })
 

